### ISSUE TITLE
Spyder's Addon Ambiance error

### ISSUE THREAD

**Issue by Hauser1** [Spearhead-Gaming-Milsim/Operation-Knockback#1]
Get an error on mission load that indicates the ambiance module can't find locations to spawn things.  Not really sure that this is all about and everything should be put in the spawn lines correctly.

**Comment by coachwn** [comment]
Yeah I removed that module as I never could get it to work correctly. Spyder has basically stopped working on it other than major bug fixes. I talked with him and Incontineia about their mods and they are working on something together now to see if it will work.

I will take a look at the mission file tonight and see what I can coax out of it.
